proxy/chart/README.md

```diff
@@ -0,0 +1,66 @@
+# OpenLineage Proxy [Helm Chart](https://helm.sh)
+
+Helm Chart for [OpenLineage Proxy](https://github.com/OpenLineage/OpenLineage/tree/main/proxy).
+
+## TL;DR
+
+```bash
+helm install ol-proxy . --dependency-update
```

Comment:
I'd be more verbose with the naming here - `openlineage-proxy`

Reply:
I'm just super lazy and default to `ol`. But, yeah verbose is the way to go 👍